### DISCUSSION TITLE
Some customization questions

### DISCUSSION THREAD

**Discussion by 3llerin** [Godiesc/firefox-one#39] [General]
I want to know if i can change firefox toolbar button icon in top left and how to do it, The facebook messenger extension is not available for download also, any alternative? And lastly it is possible to maintain the default tabs style while using firefox one default colors disabled, i like the tabs design but cant change the rest of the ui colors then, currently i use firefox color.

**Comment by Godiesc** [comment]
hi,

> I want to know if i can change firefox toolbar button icon in top left and how to do it

if you mean the Hamburger menu logo you can do it, for example you can use the file `one_tricks.css` and the code below to make that change:

```
/* Logo Hamburguer Menu */

:root:not([chromehidden~="toolbar"]) #PanelUI-menu-button .toolbarbutton-badge-stack {
        list-style-image: url("../icons/milogo.svg") !important;
}
```

I didn't create a 'Facebook messenger' extension cause the extension existed in the Firefox Store, maybe in a future I create one.


> And lastly it is possible to maintain the default tabs style while using firefox one default colors disabled

mm, I thing its not possible, the theme was made that way, maybe you should try this extension for easy change of some colors, was made for me: [GX Theme Styles](https://addons.mozilla.org/es/firefox/addon/gx-theme-styles/)

**Reply by 3llerin** [reply]
Thank you so much!! it is nearly perfect now for me, notifications on the whatsapp extension is another thing left but i think is not possible, the logo code worked perfectly!

**Reply by 3llerin** [reply]
The extension of facebook messenger i was talking about is this one in the attached files, it is out of the mozilla extensions web for some reason but it works still ok, you have to install it standalone. I have to uploaded with a .zip extension just remove that.
[xrst.vardakis@gmail.com.xpi.zip](https://github.com/user-attachments/files/16353331/xrst.vardakis%40gmail.com.xpi.zip)
